### ISSUE TITLE
Kustomize Patch Support for coredns-operator base manifests

### ISSUE THREAD

**Issue by stealthybox** [kubernetes-sigs/addon-operators#12]
Users should be able to include arbitrary patches to operator resources.
One use-case would be to mutate the Service to annotate it for an ingress-controller.

@johnsonj is working on implementing Kustomize style patch support:

https://github.com/justinsb/addon-operators/pull/1
https://github.com/kubernetes-sigs/kubebuilder-declarative-pattern/pull/44

We should get the operator side of things merged into #7 either in justin's repo before merge.
We can also just re-open the PR on this repo which unblocks merging #7.

**Comment by dholbach** [comment]
@johnsonj In https://github.com/kubernetes-sigs/addon-operators/pull/18 you say that it updates this issue. Could we break down what needs to be done here?

**Comment by johnsonj** [comment]
I think this should be closed at this point. There's lots of ways forward for patching but I believe the exploratory CL covers this issue.

**Comment by dholbach** [comment]
Super, thanks @johnsonj!
